lib/index.js

```diff
@@ -303,9 +303,6 @@ export function createUnifiedLanguageServer({
           }
         }
 
-        // This presumably should not occur: a file outside a workspace.
-        // So ignore the file.
-        /* c8 ignore next */
```

Comment:
Probably fine as part of this PR.
Though it feels a bit related to https://github.com/unifiedjs/unified-language-server/pull/52?

Reply:
No, it’s not related. If this was added as part of #52, CI would fail.

This PR adds a test and fix for the situation that according this comment should not occur, but it apparently users are running into it in practice.